ios_ui/MovieRec/MoviesToRate.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  MoviesToRate.swift
+//  MovieRec
+//
+//  Created by Yujin Cho on 8/18/17.
+//  Copyright © 2017 Yujin Cho. All rights reserved.
+//
+
+import os.log
+import UIKit
+
+class MoviesToRate {
+    
+    var userId: String?
+    var movies = [Movie]()
+    
+    init() {
+        setUserID()
+        loadMovies()
+        //print(movies.map {(movie:Movie) -> String in movie.title})
+        print("Movie Count: \(movies.count)")
+    }
+    
+    func count() -> Int {
+        return movies.count
+    }
+    
+    func downloadMoviesToRate(ratings: Ratings) {
```

Comment:
In general try to check if any of these functions and computed properties are needed outside the scope of this file, if they aren't you should make all the methods private/fileprivate.